CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing to KubeVirt-Ansible
+
+## Intro
+
+In addition for allowing deployment of KubeVirt via Ansible,
+the KubeVirt-Ansible project is open for any contributions from
+Additional projects which aims to integrate with KubeVirt, such 
+as a specific storage or network project.
+
+The integration will be in the form of Ansible roles per component
+which might later be integrated into a main Ansible Playbook Bundle.
+
+Information on how to contribute will follow in the next sections.
+
+## Contributing new Ansible roles  
+
+Contributing to KubeVirt-Ansible should be as simple as possible. 
+Have a question? Want to discuss something? Want to contribute something? Just open an
+[Issue](https://github.com/kubevirt/kubevirt-ansible/issues) or a [Pull
+Request](https://github.com/kubevirt/kubevirt-ansible/pulls).
+
+### Roles Structure
```

Comment:
Maybe we should provide an outline of the role structure before we describe how to contribute a role.  What do you think?

Reply:
Yea, I've mentioned it in the Todo:

* Add info on role template and how to role for each supported flow ( K8S,OpenShift, W/ VMs )
* Add example from existing role for KubeVirt

Do you think its best to put it before in its own section? before the contribute role section. makes sense.

Reply:
I view that an a section that introduces the structure of the repo that you can look at before you dive into how to contribute.  I think it should be very brief and the following sections can get more into the details.